### DISCUSSION TITLE
MSSQL JDBC driver, make +1h on my UTC date

### DISCUSSION THREAD

**Discussion by KevinGirard** [microsoft/mssql-jdbc#2376] [Q&A] [answered]
Hello,

I'm using JDBC driver with my ELT Semarchy xDI and I had an issue during the change time. My process retrieve 2 column date one UTC the other local (Paris) form the source Microsoft SQL Database. I have no issue with the local date column while the hour are jumping from 1h59 to 3h00. Unfortunatelly, it's not the same for the UTC date column, the driver adds +1h to each time between 2h and 3h.
This caused a lot of issue in my datawarehouse.
Is there a solution for locking this implicit modification? Do I need to convert my column into string?
Here is my actual connection string: "jdbc:sqlserver://servername:1433;databaseName=XXX;ApplicationIntent=ReadOnly;encrypt=false"
![2024-04-03_09-48-12](https://github.com/microsoft/mssql-jdbc/assets/60687649/7a424d99-5a95-4a34-9d56-394252fe239c)
![2024-04-03_09-49-21](https://github.com/microsoft/mssql-jdbc/assets/60687649/62f04db5-60bd-4a42-80a6-d7f749949231)

**Comment by barryw-mssql** [comment, marked as answer]
Hi Kevin,

It sounds like you are reporting a driver defect. As such we are requesting that you move this discussion and create a GIT issue. Please include more details including driver version, SQL Server version, problem SQL statement, etc. These details will help us more quickly identify and hopefully resolve the problem

Thx,
Barry

**Comment by Jeffery-Wasty** [comment]
Just to add onto this, we'll be closing discussions moving forward. It really helps us having everything be in one place, that being our issues page.
